warehouse/forklift/legacy.py

```diff
@@ -649,12 +681,27 @@ def file_upload(request):
         else:
             field_name = sorted(form.errors.keys())[0]
 
+        if field_name in form:
+            if form[field_name].label.text:
+                error_message = (
+                    "{value!r} is an invalid value for {field}. ".format(
+                        value=form[field_name].data,
+                        field=form[field_name].label.text) +
+                    "Error: {} ".format(form.errors[field_name][0]) +
+                    "see "
```

Comment:
I think you'll need to add something before "see" here, as there isn't a space between it and the error string. I'd recommend changing this to `", see"`.

Also, it looks like there are still newlines in the tests, as some are failing.

Reply:
Yes, I'm changing the tests, I'm going to add those fixes to the test to the same commit as to stop the commit spamming I'm already doing haha.

As to the "see" part, I do have a space after the `{}`